cibuild.sh

```diff
@@ -61,7 +62,7 @@ run_msbuild()
     
     for i in `seq 1 $RETRY_COUNT`
     do
-        o=$(mono packages/Microsoft.Build.Mono.Debug.14.1.0.0-prerelease/lib/MSBuild.exe /v:m /p:SignAssembly=false /p:DebugSymbols=false "$@")
+        o=$(mono $MONO_ARGS packages/Microsoft.Build.Mono.Debug.14.1.0.0-prerelease/lib/MSBuild.exe /v:m /p:SignAssembly=false /p:DebugSymbols=false "$@")
```

Comment:
I think we should discard this output capture. If the build fails I find it very useful to be able to look at the output and see why. Also, when the output is captured the build just sits there "Compiling CrossPlatform.sln" and I have no idea how far through it's gotten.